components/aws/sagemaker/tests/integration_tests/component_tests/test_train_component.py

```diff
@@ -11,9 +11,21 @@
     "test_file_dir",
     [
         pytest.param(
-            "resources/config/simple-mnist-training", marks=pytest.mark.canary_test
+            "resources/config/simple-mnist-training",
+            marks=pytest.mark.canary_test
+        ),
+        pytest.param(
+            "resources/config/fsx-mnist-training",
+            marks=pytest.mark.fsx_test
+        ),
+        pytest.param(
+            "resources/config/sample-pipeline-training",
+            marks=pytest.mark.canary_test,
+        ),
+        pytest.param(
+            "resources/config/spot-sample-pipeline-training",
+            marks=pytest.mark.canary_test,
```

Comment:
remove canary marker? and lets have only this test - spot-sample-pipeline-training. I think that would be sufficient ?

all other changes looks good

Reply:
done

Reply:
remove canary marker
```
pytest.param(
            "resources/config/fsx-mnist-training",
            marks=pytest.mark.fsx_test
        ),
        "resources/config/spot-sample-pipeline-training"
```